CONTRIBUTING.md

```diff
@@ -81,11 +81,20 @@ ansible-test units --docker -v --python 3.8 tests/unit/plugins/modules/net_tools
 
 The following commands show how to run integration tests:
 
+#### In docker
+
 ```.bash
-# Run integration tests for the interfaces_files module in a Docker container using the
-# fedora35 operating system image (the supported images depend on your ansible-core version):
-ansible-test integration --docker fedora35 -v interfaces_file
+# Run integration tests in a Docker container for a specific module, e.g. community.general.pacman:
+ansible-test integration -v --docker default pacman
```

Comment:
The default container is almost **never** a good idea to use for integration tests.

**Especially** for the pacman module, there's exactly one single container that's useful (and that's not even included in ansible-test's list of containers). So this example is not very helpful :)